cg/meta/workflow/balsamic.py

```diff
@@ -451,9 +463,8 @@ def family_has_correct_number_tumor_normal_samples(self, case_id: str) -> bool:
 
     def get_analyses_to_clean(self, before_date: dt.datetime = dt.datetime.now()) -> list:
         """Retrieve a list of analyses for cleaning created before certain date"""
-        analyses_before = self.store.analyses(before=before_date)
-        analyses_to_clean = self.store.analyses_to_clean(pipeline="Balsamic")
-        return [x for x in analyses_to_clean if x in analyses_before]
+        analyses_to_clean = self.store.analyses_to_clean(pipeline="Balsamic", before=before_date)
```

Comment:
```suggestion
        analyses_to_clean = self.store.analyses_to_clean(pipeline="balsamic", before=before_date)
```
Lets stick to small letters and try to have a restricted language. This is being adressed in #805